### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by zhangyuanyang21** [SubmissionsIn/DEMVC#7]
hello ！ Where is the loss function in the code ？

**Comment by SubmissionsIn** [comment]
for v in range(view):
        view_shapes.append(x[v].shape[1:])
        Loss.append('kld')
        Loss.append('mse')
        Loss_weights.append(args.lc)
        Loss_weights.append(args.Idec)

**Comment by zhangyuanyang21** [comment]
What does this code mean?

feature_model = Model(self.model.input[self.flag - 1],
                                              self.model.get_layer(name=view_name).output)

**Comment by zhangyuanyang21** [comment]
Why did the pre-training only train autoencoder and not encoder  ?
        self.autoencoder.fit(x, x, batch_size=batch_size, epochs=epochs, callbacks=cb, verbose=verbose)

**Comment by SubmissionsIn** [comment]
？？？

**Comment by zhangyuanyang21** [comment]
> ？？？

为什么预训练里只训练了autoencoder ，没训练encoder呢？

**Comment by SubmissionsIn** [comment]
autoencoder包括了encoder

**Comment by zhangyuanyang21** [comment]
你好！我在你代码的基础上加了几层新的网络，也要先放到在预训练里训练吧？

**Comment by zhangyuanyang21** [comment]
你好！我在你代码的基础上加了几层新的网络，也要先放到在预训练里训练吧？

**Comment by zhangyuanyang21** [comment]
你好，我想进一步改进您的网络结构，但是不知道怎么把我定义的网络结构嵌入您的代码里，您可以给我提供一些帮助吗？

**Comment by zhangyuanyang21** [comment]
> autoencoder包括了encoder

谢谢您的回复

**Comment by SubmissionsIn** [comment]
这个问题和具体的方法很相关，建议先熟悉整个框架

**Comment by zhangyuanyang21** [comment]
框架我看懂了，主要是训练部分看不明白是怎么操作的
